packages/compass-schema/src/stores/store.js

```diff
@@ -195,13 +195,6 @@ const configureStore = (options = {}) => {
 
     onSchemaSampled() {
       this.geoLayers = {};
-
-      process.nextTick(() => {
-        this.globalAppRegistry.emit('compass:schema:schema-sampled', {
-          ...this.state,
-          geo: this.geoLayers,
-        });
-      });
```

Comment:
The last consumer of this event was removed in 953d9b213bebfeae67c70eb2a01ee6aabd57c3ad which removed the legacy metrics package. We have tracking code for this event next to the call site of this method.